index.html

```diff
@@ -543,6 +543,99 @@ <h6>JWT Decoding</h6>
       </section>
     </section>
 
+    <section id="production">
+      <h2>Production</h2>
+
+      <p>
+        This section describes how to produce a VC-JWT encoded
+        <code>VerifiableCredential</code> from a <code>Credential</code>.
+      </p>
+
+      <p class="advisment">
+        There are currently 2 competing solutions to this problem. It is a goal
+        of the v2 work to define a breaking change to the specification that
+        reduces production rules to a single, simple, set of instructions that
+        any implementer can easily meet if they posses a software library
+        supporting [[RFC7515]] or [[RFC7519]].
+      </p>
+
+      <section>
+        <h3>In addition to...</h3>
+
+        <p>
+          There are several members (claims) of the
+          <code>Credential</code> which will need to be translated to their JOSE
+          form, and included next to the <code>vc</code> or
+          <code>vp</code> member in the <code>payload</code>.
+        </p>
+
+        <p>
+          If a member is not present in the <code>Credential</code> it MUST NOT
+          be present in the <code>VerifiableCredential</code> as either a claim
+          in the payload or a claim in the <code>vc</code> attribute of the
+          payload.
+        </p>
+
+        <p>
+          We start with an empty header, and payload objects, and we add members
+          to the header and the payload based on the content in the
+          <code>Credential</code>.
```

Comment:
which members are added to the header based on the content of the Credential..?

Reply:
afaik, none right now. 

But since the header is singed over, we must define complete construction of it to satisfy production rules.

I suspect we will define some additional "security meta data" that is also an argument of product rules.... `privateKeyJwk` being an example... which might include `kid` and `alg` which would go in the header.